### PR TITLE
[ACS-3967] added open sans font to ACA

### PR DESCRIPTION
**Pull request by SheenaMalhotra182**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

Implemented Font Styling as per [ACS-3967](https://alfresco.atlassian.net/browse/ACS-3967) Jira Ticket. This involves Updating the font style in ADW to reflect the new font "Open Sans".

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://alfresco.atlassian.net/browse/ACS-3967

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -111,6 +111,7 @@
             "styles": [
               "app/src/assets/fonts/material-icons/material-icons.css",
               "app/src/assets/fonts/muli/muli.css",
```
Comment (DenysVuika) [inline angular.json:113]: previous font probably needs to be removed?

Comment (SheenaMalhotra182) [reply angular.json]: removed unused fonts from angular.json and assets folder.

**Review by DenysVuika — APPROVED** [review]

**Comment by VitoAlbano** [comment]
Could we double check with content-ee this changes? As far as i remember we use the font reference there as well and this change might break the fonts app

**Review by VitoAlbano — CHANGES_REQUESTED** [review]
check the comment

**Comment by SheenaMalhotra182** [comment]
raised another PR for removing ‘Muli’ configurations from content-ee. Now Header, Footer, and Side Nav sections reflect the 'Open Sans' font.
 https://github.com/Alfresco/alfresco-apps/pull/4540

**Review by VitoAlbano — APPROVED** [review]

**Pull request merged**